templates/server/puppet.conf.main.erb

```diff
@@ -0,0 +1,13 @@
+## Server config
+
+    # Puppet 3.0.x requires this in both [main] and [master] - harmless on agents
+    autosign       = <%= scope.lookupvar('::puppet::autosign') %>
```

Comment:
@GregSutcliffe the original change came from one of your commits, do you remember any PuppetLabs ticket or similar? This would also influence #381.